src/core/config.py

```diff
@@ -4,7 +4,7 @@
 
 
 class Config(object):
-    REDIS_URL = os.getenv("REDIS_URL")
+    REDIS_URL = os.getenv("REDIS_URL", "redis://redis")
```

Comment:
not sure if it is a good idea to add a default value here. since the Redis database is a requirement, it should be defined by the user and when it is not, an error should be thrown.

Reply:
Oh this was some obsolete debug code - appearently this line is never used. The any other place I could find was https://github.com/SK-CERT/Taranis-NG/blob/main/src/core/managers/sse_manager.py#L11 which accesses the Environment variable again.

Reply:
I will revert the change for now

Reply:
I think Redis might be used for SSE. Take a look here, it is required to be in the config: https://pypi.org/project/Flask-SSE/